apps/trustlab/payload.config.ts

```diff
@@ -58,7 +59,7 @@ export default buildConfig({
       globals: ["site-settings"],
     },
   },
-  collections: [Media, Pages, Posts, Users] as CollectionConfig[],
+  collections: [Media, Pages, Posts, Users, Partners] as CollectionConfig[],
```

Comment:
Lets keep these sorted alphabetically (unless this sorting affects how they appear in the dashboard; if so lets add a comment so that it's clear to everyone that may edit this line in the future).